contracts/script/deploy/AlignedLayerDeployer.s.sol

```diff
@@ -86,7 +88,8 @@ contract AlignedLayerDeployer is ExistingDeploymentParser {
             ".permissions.initalPausedStatus"
         );
 
-        pauser = address(eigenLayerPauserReg);
+        pauser = address(eigenLayerPauserReg); //TODO: Change to the pauser
```

Comment:
just wondering if this is still needed, due to the TODO comment

Reply:
> just wondering if this is still needed, due to the TODO comment

I left this comment because the deployer is setting the EigenLayer address as Pauser of our contracts. I want to fix this bug in another PR

Reply:
This PR solves this TODO https://github.com/yetanotherco/aligned_layer/pull/501